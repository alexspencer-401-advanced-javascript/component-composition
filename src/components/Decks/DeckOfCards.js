import React from 'react';
import PropTypes from 'prop-types';
import Deck from './Deck.js';

const DeckOfCards = ({ props }) => {  

  return (
    <>
      <Deck title={'Characters'} props={props}/>
      <Deck title={'Gifs'} props={props}/>
    </>
  );
};

DeckOfCards.propTypes = {
  props: PropTypes.arrayOf(PropTypes.shape({
    _id: PropTypes.string,
    name: PropTypes.string,
    image: PropTypes.string,
    gifLink: PropTypes.string
  })),
};

export default DeckOfCards;
