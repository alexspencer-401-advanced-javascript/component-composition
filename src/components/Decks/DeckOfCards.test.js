import React from 'react';
import { shallow } from 'enzyme';
import DeckOfCards from './DeckOfCards';

describe('DeckOfCards component', () => {
  it('renders DeckOfCards', () => {
    const wrapper = shallow(<DeckOfCards />);
    expect(wrapper).toMatchSnapshot();
  });
});
