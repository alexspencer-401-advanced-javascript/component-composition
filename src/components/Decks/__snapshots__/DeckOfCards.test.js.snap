// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeckOfCards component renders DeckOfCards 1`] = `
<Fragment>
  <Deck
    title="Characters"
  />
  <Deck
    title="Gifs"
  />
</Fragment>
`;
